lib/main.dart

```diff
@@ -32,6 +33,7 @@ class App extends StatelessWidget {
         providers: [
           BlocProvider(create: (_) => ThemeCubit()),
           BlocProvider(create: (_) => FavoritesCubit()),
+          BlocProvider(create: (_) => FriendsBadgeCubit()),
```

Comment:
I think we can move this BlocProvider into the FriendsPageBadge since it's only needed there